kolibri/auth/migrations/0003_auto_20170309_1853.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2017-03-09 18:53
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kolibriauth', '0002_auto_20170208_0110'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='collection',
+            name='kind',
+            field=models.CharField(choices=[(b'facility', 'Facility'), (b'classroom', 'Classroom'), (b'learnergroup', 'Learner group')], max_length=20),
```

Comment:
Seems to be adding validation to this CharField, so that the only permissible options are those in this list.